### DISCUSSION TITLE
成功在甲骨文服务器的宝塔面板搭建

### DISCUSSION THREAD

**Discussion by imbalaomao** [AlistGo/alist#921] [Show and tell]
### 搭建效果

![](https://3o.hk/images/2022/04/08/QQ20220408153934.png)
配合免费域名+CF免费解析，一路白嫖是真的开心！

### 搭建过程

在宝塔中新建一个站点（以 “alist.abc.xyz” 为例） —— 不创建数据库，不创建FTP，PHP为静态 —— 查看 **[手动安装](https://alist-doc.nn.ci/docs/install/manual)** 中的指示，下载文件并解压到站点根目录，若为Linux系统则赋予755权限 —— 宝塔软件商店中下载【Supervisor管理器】并打开 —— 添加守护进程：运行目录（站点根目录）、启动命令（`./alist-linux-amd64`）、启动用户（root） —— 宝塔中设置反向代理：代理名称（随意）、目标URL（`http://127.0.0.1:5244`）。

上述操作做完后，访问站点就可以正常打开alist管理面板了

### 查询密码

在宝塔终端中输入：`sudo -i && cd /www/wwwroot/你的站点域名 && ./alist-linux-amd64 -password` 就能看到了

**Comment by mangic** [comment]
请问宝塔搭建完成后如何更新alist呢？

**Reply by imbalaomao** [reply]
下载新的alist执行文件替换原文件即可，不影响运行配置

**Reply by Catalina1219** [reply]
请问宝塔面板搭建再加上CF的cdn  怎么加https

**Reply by imbalaomao** [reply]
> 请问宝塔面板搭建再加上CF的cdn 怎么加https

可以只使用CF的https，也可以使用宝塔面板配合CF的token一键SSL

**Comment by Actuallymax** [comment]
这样反代以后，网页点击potplayer无法播放，网页自己可以播放

**Comment by qifeibro** [comment]
宝塔是怎么安装在甲骨文里的呀？
